### DISCUSSION TITLE
Which famous speech is just 272* words long?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#918] [Galaxy Trivia] [answered]
**Bonus:** What day of the week was the speech given?

**Comment by peterbaccaro** [comment, marked as answer]
The famous speech that is just 272 words long is known as the **Gettysburg Address**. It was delivered by President Abraham Lincoln on **Thursday** November 19 1863, during the American Civil War. The speech was given at the dedication of the Soldiers' National Cemetery in Gettysburg, Pennsylvania, after a major battle took place there.
